source/assets/scripts/main.js

```diff
@@ -19,7 +19,12 @@ export const myStorage = window.localStorage;
 export const numRecipeCards = { display: 0};
 //ensure there is a shopping list in storage
 if(!myStorage.getItem("SHOPPING_LIST")){
-  myStorage.setItem("SHOPPING_LIST", "{}");
+  const intialShoppingListObj = {}
+  intialShoppingListObj["Personal Shopping List"] = {
+    PB : false,
```

Comment:
Do we want to keep the default objects here?

Reply:
I think we should delete them

Reply:
the actual PB & J part should be removed. that was for testing. but `const initialShoppingListObj = {}` and `initialShoppingListObj["Personal Shopping List"]= {}` must be kept. otherwise `localStorage` may not have these keys

Reply:
i guess we could these lines of code somewhere else, but i like having them there as a reminder of the storage structure

Reply:
I see I see, but for the user it would be odd to see those things. 
Wanna leave it as a comment right below?

Reply:
the user wont see these unless they are looking directly as `main.js` or `localStorage`

Reply:
mb, miscommunication, deleting the PB&J part was what I meant~